builder/config.go

```diff
@@ -286,6 +289,7 @@ func (config *MixConfig) legacyParse(filename string) error {
 		{`^LOCAL_BUNDLE_DIR\s*=\s*`, &config.Mixer.LocalBundleDir, false},
 		{`^LOCAL_REPO_DIR\s*=\s*`, &config.Mixer.LocalRepoDir, false},
 		{`^LOCAL_RPM_DIR\s*=\s*`, &config.Mixer.LocalRPMDir, false},
+		{`^DOCKER_IMAGE_PATH\s*=\s*`, &config.Mixer.LocalRPMDir, false},
```

Comment:
Copy-pasta error?

Reply:
got lost in my poorly-done rebase effort. Should be `&config.Mixer.DockerImgPath`

Reply:
*nods*

Reply:
Finally pushed a fix of this. Sorry for the delay.